src/api/user/jest.setup.js

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
```

Comment:
You've already done this code in jest.config.js, it doesn't need to be repeated.